script.js

```diff
@@ -96,10 +82,32 @@ function updateCards(columnIndex, cards) {
 
     const h = cardTemplate.content.querySelector("h5");
     const p = cardTemplate.content.querySelector("p");
+    const prioritySpan = cardTemplate.content.querySelector(".priority");
+    const deadlineSpan = cardTemplate.content.querySelector(".deadline");
+
+    // Traducir la prioridad a español
+    let translatedPriority = card.priority;
+    if (card.priority === "High") {
+      translatedPriority = "Alta";
+    } else if (card.priority === "Medium") {
+      translatedPriority = "Media";
+    } else if (card.priority === "Low") {
+      translatedPriority = "Baja";
+    }
```

Comment:
Podes hacer algo asi:

```suggestion
   const cardTranslations = {
    "High": "Alta",
    "Medium": "Media"
    "Low": "Baja"
    };
    
    const translatedPriority = cardTranslations[card.priority];
```